doc/release-notes/0.46/0.46.md

```diff
@@ -0,0 +1,156 @@
+<!--
+* Copyright (c) 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# Eclipse OpenJ9 version 0.46.0 release notes
+
+These release notes support the [Eclipse OpenJ9&trade; 0.46.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.46.0/plan).
+
+## Supported environments
+
+OpenJ9 release 0.46.0 supports OpenJDK 8, 11, 17, 21, and 22.
+
+All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests provided by Adoptium.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.46.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.46/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr>
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk22/pull/42">#42</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk21/pull/140">#140</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/337">#337</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/743">#743</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/766">#766</a></td>
```

Comment:
None of these changes are added in this 0.46 release, they should be removed from this table. Related documentation for these should be in the user guide, a migration guide, or they can be moved to known issues. The extended Hot Code Replace deprecation notice should remain in known issues.

Reply:
Sorry, never mind.